host/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+
+add_custom_command(
+  OUTPUT ortoa_u.h ortoa_u.c ortoa_args.h
+  DEPENDS ${CMAKE_SOURCE_DIR}/ortoa.edl
+  COMMAND
+    openenclave::oeedger8r --untrusted ${CMAKE_SOURCE_DIR}/ortoa.edl
+    --search-path ${OE_INCLUDEDIR} 
+    --search-path ${OE_INCLUDEDIR}/openenclave/edl/sgx)
+
+
+add_executable(ortoa-host
+               host.cpp 
+               redis.h
+               redis.cpp
+               ../constants/constants.h
+               ${CMAKE_BINARY_DIR}/gen-cpp/RPC.h
```

Comment:
Reference to comment https://github.com/ySteinhart1/ORTOA/pull/17/files#r1367831540